python/FACTOR.py

```diff
@@ -535,7 +535,7 @@ def compute_raw_hash(self):
     #TODO: replace and raise an error if no scriptPubKey is given for master branch.
     def mine(self, mine_latest_block = True, coinbase_message = "", scriptPubKey = None ):
         global primorial_base, siever, SIEVE_MAX_LEVEL
-        self.Count = 0
+        
```

Comment:
You need to leave this here. This count should be initialized to zero every time we enter this function. If it is not globally defined line 620 will throw an error, and even if it is the printed value performed in line 620 will be wrong. So, initialize to zero here.

Reply:
@daedalus